src/libraries/System.Data.Common/src/System/Xml/DataPointer.cs

```diff
@@ -64,10 +61,10 @@ private DataRow Row
         {
             get
             {
-                XmlBoundElement rowElem = GetRowElement();
+                XmlBoundElement? rowElem = GetRowElement();
                 if (rowElem == null)
                 {
-                    return null;
+                    return null!;
```

Comment:
This will cause a NRE. We should make the property return a `DataRow?` instead I think.

Reply:
Fixed

Reply:
Why is this `!` needed?

Reply:
Because the smart person that I am I only changed the return type of the property :worried:

Fixed